### PR TITLE
[eQSL] Default QSLMSG by station profil (renew)

### PR DESCRIPTION
**Pull request by abarrau**
Hello, 
This PR is the same like PR #2694 (but previous PR was broken after a mistake of mine). 
No change on the developpement.

### CODE REVIEW

File: application/models/User_options_model.php [hunk 1]
```diff
@@ -16,16 +16,32 @@ public function set_option($option_type, $option_name, $option_array) {
...
 	}
 
-	public function get_options($option_type) {
+	public function get_options($option_type, $option_array=null) {
 		$uid=$this->session->userdata('user_id');
-		$sql='select option_name,option_key,option_value from user_options where user_id=? and option_type=?';
-		return $this->db->query($sql, array($uid, $option_type));
+		$sql_more = "";
+		$array_sql_value = array($uid, $option_type);
+		if (is_array($option_array)) {
```
[the remark this answers is not in the document]
Comment (abarrau) [reply application/models/User_options_model.php:23]: hello Joerg,
(no change from the previous PR)
This model offers various ways to perform the query on the "user options" table.
Either we use the table with the elements we need to bring out 1 specific piece of data; either the array is empty and we return a set of results.
In my case I only need 1 piece of data, so doing a fetch on the php side is more time-consuming than an operation on the database side.

[the remark this answers is not in the document]
Comment (int2001) [reply application/models/User_options_model.php:23]: Roger. it's a useful extension and doesn't destroy anything. Will approve this

**Review by int2001 — APPROVED** [review]

**Comment by HB9HIL** [comment]
I'll check this evening. On the road atm

**Review by HB9HIL — APPROVED** [review]
tested in SFLE and approved :)

**Review by HB9HIL — APPROVED** [review]
tested in SFLE and approved :)

**Pull request merged**